### PR TITLE
Revert "MWPW-152283: [TEST] switch WCS to acom domain"

### PR DESCRIPTION
**Pull request by yesil**
Reverts adobecom/milo#2504

The PR #2504 brought a commerce feature that wasn't ready to be merged into Milo due to missing accompanying Milo logic. Team has decided to revert it.

Resolves: https://jira.corp.adobe.com/browse/MWPW-153808

**Test URLs:**

- Before: https://main--milo--adobecom.hlx.page/de/drafts/mili/promo-placeholders?martech=off
- After: https://revert-2504-MWPW-152283--milo--adobecom.hlx.page/de/drafts/mili/promo-placeholders?martech=off

CC: 
- Before: https://main--cc--adobecom.hlx.page/de/drafts/mili/commerce?martech=off
- After: https://main--cc--adobecom.hlx.page/de/drafts/mili/commerce?martech=off&milolibs=revert-2504-MWPW-152283

### CODE REVIEW

**Review by bozojovicic — APPROVED** [review]

**Comment by afmicka** [comment]
@yesil @npeltier This PR also contains changes from https://github.com/adobecom/milo/pull/2529 that is merged to stage, but not yet to main and is queued in https://github.com/adobecom/milo/pull/2528.

I am not sure if anything else went in as well. I am not seeing anything broken and the issue with the tax label is being fixed with this PR.

**Review by robert-bogos — APPROVED** [review]

File: libs/deps/commerce.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
...
-  //# sourceMappingURL=commerce.js.map
-  
\ No newline at end of file
+// branch: develop commit: b3f6608faa10db8d0187b310044d4690d063f1bf Thu, 06 Jun 2024 16:47:00 GMT
```
Comment (narcis-radu) [inline libs/deps/commerce.js:1]: @yesil - I see this file generated from different branches and I'm wondering if there's any risk of introducing unwanted changes. Should the file be built from an official branch like `main` after everything has been properly tested? cc @npeltier , @3ch023

**Review by overmyheadandbody — APPROVED** [review]
It seems this is just a revert, so it should not introduce issues; however, to @narcis-radu 's point, the commerce logic is sometimes quite opaque and it's difficult to follow.

Why was this merged in the first place, since it lacked an important piece of the logic? Does your team have assigned QA resources looking into PRs before they get merged to Stage?

**Comment by npeltier** [comment]
@overmyheadandbody @narcis-radu 
big part of milo commerce code base has Web Components in their core. They are developed separately than milo, now in [github.com/adobecom/mas](http://github.com/adobecom/mas). For now code inclusion in milo is to copy over built libraries to milo codebase, and eventually do some extra specific milo magic. This means a complicated game of doing mas PR then milo PR when all is good. We've been trying to deal with that problem as much [as we could](https://github.com/orgs/adobecom/discussions/2378) but MAS project came in, and we rather go to change the process which is direct consumption of mas artifacts (that includes our own pipeline & test & all) that will be needed anyway, and should help smoothening things around.

File: libs/deps/commerce.js (hunk 1 above)
Comment (yesil) [reply libs/deps/commerce.js:1]: Narcis, I opened this revert PR via github, and it should normally revert to previous good version of commerce.js that was in prod.

**Comment by yesil** [comment]
@narcis-radu @overmyheadandbody after another round of discussion, we decided to wait until next week to assess the situation with the business.
Currently GWP has an authoring workaround.
Therefore we are closing this PR for now, and will re-open if needed.
Sorry for the noises.

**Comment by npeltier** [comment]
re-opening as we think it will be safer getting regression fixed that way

**Review by npeltier — APPROVED** [review]

**Review by 3ch023 — APPROVED** [review]

**Pull request merged**